cmd/bucket-handlers.go

```diff
@@ -411,6 +411,11 @@ func (api objectAPIHandlers) DeleteMultipleObjectsHandler(w http.ResponseWriter,
 		return
 	}
 
+	// Convert object name delete objects if it has `/` in the beginning.
+	for i := range deleteObjects.Objects {
+		deleteObjects.Objects[i].ObjectName = trimLeadingSlash(deleteObjects.Objects[i].ObjectName)
+	}
```

Comment:
doesn't DeleteObjectHandler also need to trim slashes in the objectname?

Reply:
Those already do @poornas this is a continuation PR